docs/background/lossfunction.md

```diff
@@ -0,0 +1,19 @@
+# Loss functions
+
+A loss function compares model outputs to true targets, resulting in a loss. For a loss function to be compatible with the standard supervised training loop, the following properties must hold.
```

Comment:
```suggestion
A loss function compares model outputs to true targets and returns a loss value or error a particular set of outputs. For a loss function to be compatible with the standard supervised training loop, the following properties must hold.
```

Reply:
this "a loss value or error a particular..." doesn't flow in my head; did you mean "a loss value, or error"/"a loss value (or error)"?